workspaces/quay/plugins/quay-actions/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@backstage-community/plugin-scaffolder-backend-module-quay",
+  "description": "The quay-actions module for @backstage-community/plugin-scaffolder-backend-module-quay",
+  "version": "2.2.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "backstage": {
+    "role": "backend-plugin-module",
+    "supported-versions": "1.32.5",
+    "pluginId": "scaffolder",
+    "pluginPackage": "@backstage/plugin-scaffolder-backend"
+  },
+  "exports": {
+    ".": "./src/index.ts",
+    "./package.json": "./package.json"
+  },
+  "typesVersions": {
+    "*": {
+      "package.json": [
+        "package.json"
+      ]
+    }
+  },
+  "scripts": {
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "lint:check": "backstage-cli package lint",
+    "lint:fix": "backstage-cli package lint --fix",
+    "postpack": "backstage-cli package postpack",
+    "prepack": "backstage-cli package prepack",
+    "start": "backstage-cli package start",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "tsc": "tsc",
+    "prettier:check": "prettier --ignore-unknown --check .",
+    "prettier:fix": "prettier --ignore-unknown --write ."
+  },
+  "dependencies": {
+    "@backstage/backend-plugin-api": "^1.0.1",
+    "@backstage/plugin-scaffolder-node": "^0.5.0",
+    "@material-ui/core": "^4.12.4",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1"
+  },
+  "devDependencies": {
+    "@backstage/cli": "0.28.2",
+    "@backstage/plugin-scaffolder-node-test-utils": "0.1.14",
+    "@spotify/prettier-config": "^15.0.0",
+    "@types/react": "^18.3.12",
+    "@types/react-dom": "^18.3.1",
+    "prettier": "3.3.3"
+  },
+  "files": [
+    "dist",
+    "dist-dynamic/*.*",
+    "dist-dynamic/dist/**"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/community-plugins",
+    "directory": "workspaces/quay/plugins/quay-actions"
+  },
+  "maintainers": [
+    "@janus-idp/maintainers-plugins"
```

Comment:
this isn't meaningful outside the janus-idp org. Should this be some other value?

Reply:
I've been updating this to match the CODEOWNERS file. Eg: https://github.com/backstage/community-plugins/blob/main/workspaces/topology/plugins/topology/package.json#L109